roxie/ccd/ccdkey.cpp

```diff
@@ -1222,10 +1220,11 @@ class InMemoryIndexManager : public CInterface, implements IInMemoryIndexManager
         }
         if (preload && !loadedIntoMemory) // loaded but NOT originally seen as preload, lets try to generate keys...
         {
+            assertex((size_t)totalSize == totalSize);
```

Comment:
Wouldn't it be better to throw the same exception as below (insufficient memory to preload file) ?